api/src/main/java/org/regenstrief/linkage/io/OpenMRSReader.java

```diff
@@ -92,39 +82,13 @@ private Criteria createCriteria() {
 		createHibernateSession().clear();
 		criteria = createHibernateSession().createCriteria(Patient.class).setMaxResults(PAGING_SIZE)
 		        .setFirstResult(pageNumber * PAGING_SIZE);
-		if (projections != null) {
```

Comment:
Does this mean that we no longer need the projections class level variable?

Reply:
Yes we no longer need it, I removed them in the branch for my next PR for PTM-99 otherwise I would serious conflicts to deal with I add the change this PR.

Reply:
Please see the PTM-99 branch's commit https://github.com/wluyima/openmrs-module-patientmatching/commit/a18fd1c5ed1c6e49ca7b3084968a4bc418d28a4a